src/transformers/trainer.py

```diff
@@ -188,6 +188,10 @@
 
 if is_sagemaker_mp_enabled():
     import smdistributed.modelparallel.torch as smp
+    from smdistributed.modelparallel import __version__ as SMP_VERSION
+
+    SMP_PRE_1_10 = version.parse(SMP_VERSION) < version.parse("1.10")
+    SMP_POST_1_10 = not SMP_PRE_1_10
```

Comment:
```suggestion
    IS_SAGEMAKER_MP_POST_1_10 = version.parse(SMP_VERSION) >= version.parse("1.10")
else:
    IS_SAGEMAKER_MP_POST_1_10 = False
```
We don't need two global constants since we can easily use `not` in a test.

Reply:
Sounds good to me!